header/header_modifier.go

```diff
@@ -95,12 +102,15 @@ func modifierFromJSON(b []byte) (*parse.Result, error) {
 
 	modifier := NewModifier(msg.Name, msg.Value)
 
-	if msg.Behavior == "" || msg.Behavior == "replace" {
-		modifier.SetBehavior(ReplaceValues)
-	} else if msg.Behavior == "append" {
+	switch msg.Behavior {
```

Comment:
downcase this with `strings.ToLower(msg.Behavior)`

Reply:
Do we have any other case insensitive tokens in the Martian API? I'd argue against that sort of behavior only in one place (e.g. why make the behavior enumeration case insensitive and not the "behavior" key itself, or all the JSON keys for that matter?) especially since the API errors if the enumeration doesn't match one of the known values.

Reply:
ah ok yeah that's fair